src/registrar/assets/js/get-gov.js

```diff
@@ -232,14 +246,88 @@ function handleValidationClick(e) {
   for(const input of needsValidation) {
     input.addEventListener('input', handleInputValidation);
   }
+  const alternativeDomainsAvailability = document.getElementById('check-avail-for-alt-domains');
   const activatesValidation = document.querySelectorAll('[validate-for]');
+
   for(const button of activatesValidation) {
-    button.addEventListener('click', handleValidationClick);
+    // Adds multi-field validation for alternative domains
+    if (button === alternativeDomainsAvailability) {
+      button.addEventListener('click', (e) => {
+        handleFormsetValidationClick(e, alternativeDomainsAvailability)
+      });
+    } else {
+      button.addEventListener('click', handleValidationClick);
+    }
   }
+
+  
+  // Add event listener to the "Check availability" button
+  const checkAvailabilityButton = document.getElementById('check-availability-button');
```

Comment:
This can be removed and the removeFormErrors(input, true); could be added to the handleValidationClick method right before runValidators

Reply:
Could you elaborate on why `checkAvailabilityButton` should be removed along with the `removeFormErrors(input, true)`? I am confused bc this section seems vital to me